packages/ibm-products/src/components/Datagrid/Extensions/Filtering/Panel.stories.js

```diff
@@ -40,6 +41,16 @@ export default {
   },
 };
 
+// This is to show off the View all button in checkboxes
+const dummyCheckboxes = Array(25)
+  .fill(null)
+  .map((_, index) => ({
+    id: index,
```

Comment:
Since you're spreading these properties onto the `Checkbox` component, you end up with a prop type error here because the `id` for `Checkbox` should be `string`, but here you're setting it to a number via the `index`.